wiki/Legal/Music_Licensing/en.md

```diff
@@ -2,7 +2,7 @@
 legal: true
 ---
 
-# Music Licensing
```

Comment:
Can you also rename the folder name to `osu!_Music_Licensing`?

Reply:
oh never mind, I looked again it it seems like the others do something similar

Reply:
the rest of this directory is inconsistent with the titles in the same way

can be fixed somewhere else (I think the "osu!" prefix should be removed from everything here)

Reply:
don't think this is required

Reply:
by "everything" i mean the whole wiki cuz there are other cases of this. but sure we can do the Legal ones here

I guess the directory names should be the same as the current titles (after stripping osu!) ?